tests/test_feature_panel.py

```diff
@@ -0,0 +1,248 @@
+from app.database.models import Feature, UserFeature
+import pytest
+import app.internal.features as internal
+import app.routers.features as route
+
+
+@pytest.fixture
+def mock_features():
+    return [
+        {
+            "name": 'test',
+            "route": '/test',
+            "description": 'testing',
+            "creator": 'test'
+        }
+    ]
+
+
+@pytest.fixture
+@pytest.mark.usefixtures('session')
+def feature(session):
+    test = Feature(
+        name='test',
+        route='/test',
+        description='testing',
+        creator='test',
+        icon='test'
+    )
+
+    session.add(test)
+    session.commit()
+
+    yield test
+
+    session.query(Feature).delete()
+
+
+@pytest.fixture
+@pytest.mark.usefixtures('session')
+def association_off(session):
+    print(session)
+    test = UserFeature(
+        feature_id=1, user_id=1, is_enable=False)
```

Comment:
consider use "user" fixture and then you can pass user.id instead of "1"

Reply:
It doesn't really matter much, but I can add it.

Reply:
Please if you have questions about the features panel ask them in the features panel PR, this is a icons PR.